src/commcare_cloud/commands/deploy.py

```diff
@@ -50,15 +53,23 @@ def run(self, args, unknown_args):
                 if deploy_component != 'both':
                     _warn_no_formplayer()
                 self.deploy_commcare(environment, commcare_rev, args, unknown_args)
-
         if deploy_component in ['formplayer', 'both']:
+            if deploy_component != 'both':
```

Comment:
Nitpick: this would be clearer if it were `if deploy_component == 'formplayer'`

Reply:
I was following the convention three/four lines above 🤷

Reply:
fair enough